frontend/src/components/home/measurements/client/ClientsAndProjectsEvolution.tsx

```diff
@@ -0,0 +1,142 @@
+import { Row } from "react-bootstrap";
+import ClientsAndProjectsEvolutionChart from "./ClientsAndProjectsEvolutionChart";
+import ClientsAndProjectsEvolutionList from "./ClientsAndProjectsEvolutionList";
+import { Container } from "react-bootstrap";
+import { Col } from "react-bootstrap";
+import { useState } from "react";
+import moment, { Moment } from "moment";
+import { useSelector } from "react-redux";
+import { ConfacState } from "../../../../reducers/app-state";
+import { t } from "../../../utils";
+import { Link } from "react-router-dom";
+import PeriodPicker from "../PeriodPicker";
+
+export interface clientChartData {
+  year: number;
+  monthIndex: number;
+  month: string;
```

Comment:
JSdoc info...

- Wat is monthIndex: is dat zero based of 1 based?
- month: in welk format is dat?